packages/ocs/queries/ceph-storage.ts

```diff
@@ -319,3 +324,10 @@ export const breakdownIndependentQueryMap = (
     },
   },
 });
+
```

Comment:
this comment serves as a search keyword, so that we remember every query which needs to be updated later...
```suggestion
// ToDo (epic 4422): This should work (for now) as "managedBy" will be unique,
// but moving forward add a label to metric for CephCluster namespace and use that instead (update query).
```

Reply:
@weirdwiz these got missed... plz add this...